src/time_tagger/connection/repository.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+from enum import Enum
+
+
+class ConnectionStatusEnum(Enum):
+    SUCCESS = "success"
+    FAILED = "failed"
```

Comment:
maybe we can use a bool call is_connected or something ?

Reply:
Now I remember why I chose this enum. It was because of the logic of `get_time_tagger_name_by_connection_status()`.
I will think how to have this method more readable if we use the bollean. But seems that the correct is to use the boolean here. 
I will not resolve this conversation yet, but until the end of this PR I will back to this. :)